pages/utils.js

```diff
@@ -0,0 +1,78 @@
+import { Header } from "~/components/Markdown/Components/headers";
```

Comment:
This created a new route/page in our app. https://wpds-docs-git-feat-create-dive-deeper-feed.preview.now.washingtonpost.com/utils. You could place these utils in a new folder in /components.